edb/pgsql/resolver/expr.py

```diff
@@ -611,6 +608,8 @@ def resolve_LockingClause(
     ),
 }
 
+funcs_with_text_args: set[str] = {'json_build_object'}
```

Comment:
Ew.

Reply:
Leave my premature abstractions alone!

Maybe I should just think of a few pg functions that should end up in here and add them before they break some query and people have to open issues about it.

Reply:
Ok, this is embarrassing: there is also `jsonb_build_object` and `json_build_array` and quite a few more.